### PR TITLE
Add javadoc for SqlQueryFlow - Map and Transform - Part II

### PR DESCRIPTION
**Pull request by anthonylouisbsb**

### CODE REVIEW

**Comment by anthonylouisbsb** [comment]
Task related to the issue https://github.com/s1mbi0se/simbiose_knowledge_base/issues/631

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java [hunk 1]
```diff
@@ -132,6 +132,9 @@ public QueryManagerStats getStats()
     /**
      * Generates the next query identifier.
      *
+     * The id format is:
+     * - YYYYMMdd_HHmmss_generic-count_coordinator-id
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java]: I think we could use {} in here:
```suggestion
     * - `{YYYYMMdd_HHmmss}_{generic-count}_{coordinator-id}
```
It seems better to read

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 2]
```diff
@@ -158,13 +158,13 @@ public void stop()
...
+     * Receives POST requests that try to create new queries to be executed on the server.
      *
-     * @param statement a sql command
-     * @param servletRequest an object that provides information about the http request
+     * @param statement a SQL command
+     * @param servletRequest an object that provides information about the HTTP request
      * @param httpHeaders an object containing information about the request headers
-     * @param uriInfo an object containing metadata about the uri of endpoint
-     * @return the information about query that was created
+     * @param uriInfo an object containing metadata about the URI of endpoint
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]:
```suggestion
     * @param uriInfo an object containing metadata about the URI of the endpoint
```

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 3]
```diff
@@ -158,13 +158,13 @@ public void stop()
...
+     * @return the information about the query that was created
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: The word "Information" seems too generic. What is this information? The query metadata?
Let's try not to be too generic.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 4]
```diff
@@ -441,10 +441,10 @@ public synchronized boolean isSubmissionFinished()
          * Retrieves an object that contains all information about the dispatched query.
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: The word "Information" seems too generic. What is this information? The query metadata?
Let's try not to be so generic if possible.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 5]
```diff
@@ -513,12 +513,12 @@ private QueryResults createQueryResults(long token, UriInfo uriInfo, DispatchInf
         }
 
         /**
-         * Gets the URI where information about query can be found inside server if it is not completed.
+         * Gets the URI where information about query can be found inside the server if it is not completed.
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: The word "Information" seems too generic. What is this information? The query metadata? The query result?
Let's try not to be so generic if possible.

File: presto-main/src/main/java/io/prestosql/execution/QueryState.java [hunk 6]
```diff
@@ -67,7 +67,9 @@
     }
 
     /**
-     * Is this a terminal state.
+     * Returns if the query execution finished.
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/execution/QueryState.java]:
```suggestion
     * Returns if the query execution is finished.
```

File: presto-main/src/main/java/io/prestosql/execution/QueryState.java [hunk 7]
```diff
@@ -67,7 +67,9 @@
...
+     *
+     * @return a flag which indicates if the query execution finished
```
Comment (jpedroantunes) [inline presto-main/src/main/java/io/prestosql/execution/QueryState.java]:
```suggestion
     * @return a flag which indicates if the query execution is finished
```

**Review by jpedroantunes — CHANGES_REQUESTED** [review]

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java (hunk 3 above)
Comment (anthonylouisbsb) [reply presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: changed to **the result of the processed query**

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java (hunk 4 above)
Comment (anthonylouisbsb) [reply presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: Changed to
```
Retrieves an object that contains the result of a dispatched query.
```

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java (hunk 5 above)
Comment (anthonylouisbsb) [reply presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java]: Changed to
```
Gets the URI where a query's process result can be found inside the server if it is not completed.
```

**Review by jpedroantunes — APPROVED** [review]

**Pull request merged**